pmm/list.go

```diff
@@ -155,6 +155,8 @@ func (a *Admin) List() error {
 						f := fmt.Sprintf("%s/config/qan-%s.conf", agentBaseDir, kvp.Value)
 						querySource, _ := getQuerySource(f)
 						opts = append(opts, fmt.Sprintf("query_source=%s", querySource))
+						queryExamples, _ := getQueryExamples(f)
```

Comment:
This will work but I am wondering if later the agent will be able to handle an empty string in case of an error.
I mean, if there is an error reading the file or there is an error unmarshaling the json struct, `queryExamples` will be empty.